src/app/subnets/views/SubnetDetails/StaticDHCPLease/ReserveDHCPLease/ReserveDHCPLease.tsx

```diff
@@ -202,7 +202,11 @@ const ReserveDHCPLease = ({
           <FormikField
             cidr={subnet.cidr}
             component={PrefixedIpInput}
-            label="IP address"
+            disabled={!!reservedIpId}
+            help={
+              !!reservedIpId ? "You cannot edit a reserved IP address." : null
```

Comment:
Is is "a" reserved IP or "this" reserved IP?

Reply:
"a". Once a reservation is made, you can edit the MAC and the comment, but not the IP. This is true for all reservations